CONTRIBUTING.md

```diff
@@ -79,8 +79,10 @@ mvn -f sdk/{projectForlderDir}/pom.xml -Dgpg.skip clean install
 
 ## Versions and versioning
 
-Tooling has been introduced to centralize versioning and help ease the pain of updating artifact versions in POM and README files. Under the eng\versioning directory there version text files, one for client (version_client.txt) and one for data (version_data.txt). The format of the version files is as follows:
-groupId:artifactId;dependency-version;current-version
+Tooling has been introduced to centralize versioning and help ease the pain of updating artifact versions in POM and README files. Under the eng\versioning directory there exists version text files, one for client (version_client.txt) and one for data (version_data.txt). The format of the version files is as follows:
```

Comment:
```suggestion
Tooling has been introduced to centralize versioning and help ease the pain of updating artifact versions in POM and README files. Under the eng\versioning directory there exists version text files, one for client ([version_client.txt](./eng/versioning/version_client.txt)) and one for data ([version_data.txt](./eng/versioning/version_data.txt)). The format of the version files is as follows:
```

Reply:
I like the links because I'm not a fan of clicking around. ;(

Reply:
Thanks for the suggestion, I'll update the doc and submit another PR once I've got more feedback for changes.

Reply:
Updated with the latest PR.